src/garrysmod/gamemodes/zombiesurvival/gamemode/cl_mapvote.lua

```diff
@@ -0,0 +1,40 @@
+GM.MapVote = GM.MapVote or {}
+
+function GM.MapVote:Show()
+    if not self.Active then return end
+
+    GAMEMODE.MapVote.Panel = vgui.Create("ZSMapVote")
+    GAMEMODE.MapVote.Panel:SetEndsAt(self.EndsAt)
+    GAMEMODE.MapVote.Panel:SetMaps(self.Maps)
+
+    GAMEMODE.MapVote.Panel:SetWide(math.min(974, ScrW() * 0.65) * math.max(1, BetterScreenScale()))
```

Comment:
Magic numbers here. I'm assuming 974 is the max combined width of the panels children?

Reply:
Agreed. It is magic. I copied it from the width of the scoreboard.

https://github.com/rbreslow/zs/blob/0396e6bf875d7844cc03a624bfece64c08ae8380/src/garrysmod/gamemodes/zombiesurvival/gamemode/cl_scoreboard.lua#L12

I don't know what it stands for, but I'm not sure there's a great way to find out. When we do the great UI overhaul with Ben, we can determine a better value to use.